### ISSUE TITLE
Reorganize chapters

### ISSUE THREAD

**Issue by davidmolina** [OperationCode/eloquent-js#6]
Perhaps, 01, 02, 03, etc.

**Comment by jjhampton** [comment]
Agreed - current syntax is a bit too wordy and long.

**Comment by jjhampton** [comment]
@CodeNeophyte I merged in my branch for Chapter 3.  Whenever you want to merge your own Chapter 3 branch in, we can revise the chapter names.

I think `01`, `02`, ... is a good idea.  I've noticed that when I'm in the command line and I'm trying to `cd` to one of the current chapter directories, it's a bit of a pain to either type in the entire directory (`Chapter-03-Exercises`), and since they all start with `Chapter-...`, it makes the tab-completion not as efficient either.

**Comment by CodeNeophyte** [comment]
i just merged my branch, so i'm going to go ahead and work on renaming the chapters tomorrow. Is it good then if they just get named just 01, 02,... instead of Chapter 01 or Ch 01. Like you said, chapter and exercise are a given, so no need for it then. If no issues with this then i'll press on with it.

**Comment by jjhampton** [comment]
01, 02, etc. seem good to me, thanks!

**Comment by CodeNeophyte** [comment]
Folders have been renamed. We can close this issue.
